source/client-side-encryption/client-side-encryption.rst

```diff
@@ -2521,6 +2513,7 @@ Changelog
    :align: left
 
    Date, Description
+   22-10-05, Remove spec front matter.
```

Comment:
Changelog format is different from all others.

Reply:
Same reasoning as https://github.com/mongodb/specifications/pull/1311/files#r987580269

This and change streams were intentionally left as-is.